deploy/docs/Performance.md

```diff
@@ -5,9 +5,9 @@ For larger or more volatile loads, we recommend [enabling Fluentd autoscaling](.
 ## Recommendations
 
 1. At least **8 Fluentd-logs** pods per **1 TB/day** of logs.
-2. At least **4 Fluentd-metrics** pods per **120k DPM** of metrics.
-3. The Prometheus pod will use on average **2GiB** memory per **120k DPM**; however in our experience this has gone up to **5GiB**, so we recommend allocating ample memory resources for the Prometheus pod if you wish to collect a high volume of metrics for a larger cluster.
-4. For clusters with 500 application pods or greater, we found the following configuration changes in the [`values.yaml`](./../helm/sumologic/values.yaml) file to lead to a more stable experience:
+1. At least **4 Fluentd-metrics** pods per **120k DPM** of metrics.
+1. The Prometheus pod will use on average **2GiB** memory per **120k DPM**; however in our experience this has gone up to **5GiB**, so we recommend allocating ample memory resources for the Prometheus pod if you wish to collect a high volume of metrics for a larger cluster.
+1. For clusters with 500 application pods or greater, we found the following configuration changes in the [`values.yaml`](./../helm/sumologic/values.yaml) file to lead to a more stable experience:
```

Comment:
Pretty strange IMHO

Reply:
This violated `MD029 - Ordered list item prefix`

```
deploy/docs/Performance.md:8: MD029 Ordered list item prefix
deploy/docs/Performance.md:9: MD029 Ordered list item prefix
deploy/docs/Performance.md:10: MD029 Ordered list item prefix
deploy/docs/Performance.md:23: MD029 Ordered list item prefix
deploy/docs/Performance.md:24: MD029 Ordered list item prefix
```

Reply:
🤯